activestorage/app/models/active_storage/variant.rb

```diff
@@ -91,6 +91,14 @@ def filename
     ActiveStorage::Filename.new "#{blob.filename.base}.#{variation.format.downcase}"
   end
 
+  def content_type_for_serving
+    content_type
+  end
+
+  def forced_disposition_for_serving
+    nil
+  end
```

Comment:
Shouldn't these methods implement the same logic from `Blob`?

https://github.com/rails/rails/blob/90b55e57a91db606e91015f8a7fd22279970b498/activestorage/app/models/active_storage/blob.rb#L248-L256

What if we factored out a `ActiveStorage::Blob::Servable` module (into `activestorage/app/models/active_storage/blob/servable.rb`), and then included that in `ActiveStorage::Blob` and `ActiveStorage::Variant`?  The module could define the `content_type_for_serving` and `forced_disposition_for_serving` methods, plus some of the methods they depend on (e.g. `forcibly_serve_as_binary?` and `allowed_inline?`, though not `content_type`).

Reply:
Done 👍🏼